packages/console/src/assets/images/air-play.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="21" height="20" viewBox="0 0 21 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M11.1917 11.2083C11.1155 11.0949 11.0126 11.002 10.8919 10.9377C10.7713 10.8734 10.6368 10.8398 10.5001 10.8398C10.3634 10.8398 10.2288 10.8734 10.1082 10.9377C9.9876 11.002 9.88464 11.0949 9.80841 11.2083L6.47508 16.2083C6.40194 16.335 6.36344 16.4787 6.36344 16.625C6.36344 16.7713 6.40194 16.915 6.47508 17.0417C6.54029 17.1718 6.63846 17.2826 6.75982 17.363C6.88117 17.4435 7.02147 17.4907 7.16675 17.5H13.8334C13.9842 17.4996 14.132 17.4583 14.2611 17.3805C14.3903 17.3028 14.4959 17.1914 14.5667 17.0583C14.6337 16.928 14.665 16.7824 14.6577 16.6361C14.6504 16.4898 14.6047 16.348 14.5251 16.225L11.1917 11.2083ZM8.72508 15.8333L10.5001 13.1667L12.2751 15.8333H8.72508ZM16.3334 2.5H4.66675C4.00371 2.5 3.36782 2.76339 2.89898 3.23223C2.43014 3.70107 2.16675 4.33696 2.16675 5V12.5C2.16675 13.163 2.43014 13.7989 2.89898 14.2678C3.36782 14.7366 4.00371 15 4.66675 15H5.37508C5.5961 15 5.80806 14.9122 5.96434 14.7559C6.12062 14.5996 6.20842 14.3877 6.20842 14.1667C6.20842 13.9457 6.12062 13.7337 5.96434 13.5774C5.80806 13.4211 5.5961 13.3333 5.37508 13.3333H4.66675C4.44573 13.3333 4.23377 13.2455 4.07749 13.0893C3.92121 12.933 3.83341 12.721 3.83341 12.5V5C3.83341 4.77899 3.92121 4.56702 4.07749 4.41074C4.23377 4.25446 4.44573 4.16667 4.66675 4.16667H16.3334C16.5544 4.16667 16.7664 4.25446 16.9227 4.41074C17.079 4.56702 17.1667 4.77899 17.1667 5V12.5C17.1667 12.721 17.079 12.933 16.9227 13.0893C16.7664 13.2455 16.5544 13.3333 16.3334 13.3333H15.6251C15.4041 13.3333 15.1921 13.4211 15.0358 13.5774C14.8795 13.7337 14.7917 13.9457 14.7917 14.1667C14.7917 14.3877 14.8795 14.5996 15.0358 14.7559C15.1921 14.9122 15.4041 15 15.6251 15H16.3334C16.9965 15 17.6323 14.7366 18.1012 14.2678C18.57 13.7989 18.8334 13.163 18.8334 12.5V5C18.8334 4.33696 18.57 3.70107 18.1012 3.23223C17.6323 2.76339 16.9965 2.5 16.3334 2.5Z" fill="white" fill-opacity="0.7"/>
```

Comment:
Don't we need to modify the fill color to `currentColor`?

Reply:
Updated.